README.md

```diff
@@ -6,87 +6,66 @@ Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy an
 applications on your own servers. Deis builds on [Kubernetes](http://kubernetes.io/) to provide
 a lightweight, [Heroku-inspired](http://heroku.com) workflow.
 
-## Work in Progress
+## Beta Status
 
-![Deis Graphic](https://s3-us-west-2.amazonaws.com/get-deis/deis-graphic-small.png)
-
-Deis Builder v2 is changing quickly. Your feedback and participation are more than welcome, but be
-aware that this project is considered a work in progress.
+This Deis component is currently in beta status, and we welcome your input! If you have feedback, please [submit an issue](https://github.com/deis/builder/issues) and if you'd like to participate in development, please read the "Development" section below and [submit a pull request](https://github.com/deis/builder/pulls).
```

Comment:
fixed in https://github.com/arschles/deis-builder/commit/2ad6338aa58fcb6c7c47c86511a6b18127816ae0